Assets/Scripts/Managers/CutsceneManager.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class CutsceneManager : MonoBehaviour {
+	// Start is called before the first frame update
+	private enum CutScene {
+		Intro = 14, // Name is humanname and number is duration in seconds.
+		Church = 7,
+		Outro = 12
+	}
```

Comment:
That's NOT how you should use enums 😓

Reply:
Hoe dan wel? This is not how you submit feedback. When submitting feedback you clarify why something is wrong, this is not what has happened here. It just states what is wrong but not why. :D

Reply:
Enums are actually used to specify a unique value. Might someone need another cutscene with the same duration as one of these, that wouldn't be possible. Better would be to use a Dictionary, create a new data class with a corresponding list or just use a simple list with durations in the case of fixed cutscene IDs. Another way would be to use list of tuples, containing the name (`string`) in value 1, and the duration (`float`) in value 2.

Reply:
Fair, had alleen geen zin om t uit te zoeken hoe ookalweer een dictionary werkte voor dit simpele eenmalige probleem. hihi

Reply:
Agreed hahaha